### PR TITLE
add CopyTranslator 9.0.2

### PR DESCRIPTION
**Pull request by ivaquero**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/copytranslator.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask "copytranslator" do
+  version "9.0.2"
+  sha256 "5da337e06e1ac6325627f99a0662042b1d97689613685cc046441d41d6d1f075"
+  
+  # github.com/CopyTranslator/CopyTranslator/ was verified as official when first introduced to the cask
+  url "https://github.com/CopyTranslator/CopyTranslator/releases/download/v#{version}/copytranslator-#{version}.dmg"
+  appcast "https://github.com/CopyTranslator/CopyTranslator/releases.atom"
+  name "copytranslator"
+  desc "复制即翻译的外文辅助阅读翻译解决方案"
```
Comment (reitermarkus) [inline Casks/copytranslator.rb:9]: The description should be in English.

File: Casks/copytranslator.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cask "copytranslator" do
+  version "9.0.2"
+  sha256 "5da337e06e1ac6325627f99a0662042b1d97689613685cc046441d41d6d1f075"
+  
+  # github.com/CopyTranslator/CopyTranslator/ was verified as official when first introduced to the cask
+  url "https://github.com/CopyTranslator/CopyTranslator/releases/download/v#{version}/copytranslator-#{version}.dmg"
+  appcast "https://github.com/CopyTranslator/CopyTranslator/releases.atom"
+  name "copytranslator"
```
Comment (reitermarkus) [inline Casks/copytranslator.rb:8]:
```suggestion
  name "CopyTranslator"
```

File: Casks/copytranslator.rb (hunk 1 above)
Comment (integzz) [reply Casks/copytranslator.rb]: Sure, I just changed the description.

File: Casks/eshelper.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+cask "eshelper" do
```
Comment (reitermarkus) [inline Casks/eshelper.rb:1]: Remove this cask form this PR.

Comment (integzz) [reply Casks/eshelper.rb:1]: I am sorry for committing an extra PR accidentally.

File: Casks/copytranslator.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+cask "copytranslator" do
+  version "9.0.2"
+  sha256 "5da337e06e1ac6325627f99a0662042b1d97689613685cc046441d41d6d1f075"
+
+  # https://github.com/CopyTranslator/CopyTranslator/ was verified as official when first introduced to the cask
+  url "https://github.com/CopyTranslator/CopyTranslator/releases/download/v#{version}/copytranslator-#{version}.dmg"
+  appcast "https://github.com/CopyTranslator/CopyTranslator/releases.atom"
+  name "CopyTranslator"
+  desc "Copy-as-translation solutions for reading and translation"
```
Comment (reitermarkus) [inline Casks/copytranslator.rb:9]: What does “Copy-as-translation” mean?

Comment (integzz) [reply Casks/copytranslator.rb:9]: It is Chinglish... In Chinese version, it means real-time translation while you copy. I got this by Google Translate, I should have revised it.

Comment (reitermarkus) [reply Casks/copytranslator.rb:9]: Maybe “Translates text in real-time while copying”?

Comment (integzz) [reply Casks/copytranslator.rb:9]: Yes, that is right.

File: Casks/eshelper.rb (hunk 3 above)
Comment (reitermarkus) [inline Casks/eshelper.rb:1]: Please remove this cask from this PR.

**Comment by integzz** [comment]
Thank you very much, Markus. You are so nice.

**Review by reitermarkus — APPROVED** [review]

**Comment by reitermarkus** [comment]
Thanks, @integzz!

**Pull request merged**